lib/widgets/app.dart

```diff
@@ -117,7 +118,12 @@ class _ZulipAppState extends State<ZulipApp> with WidgetsBindingObserver {
         final initialAccountId = globalStore.accounts.firstOrNull?.id;
         return MaterialApp(
           title: 'Zulip',
-          localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+          localizationsDelegates: const [
+            ...ZulipLocalizations.localizationsDelegates,
+            GlobalMaterialLocalizations.delegate,
```

Comment:
Interesting, yeah, this looks like probably a needed change.

Just to confirm my understanding: what happens if you leave this change out?

Let's put this change in a commit of its own, perhaps at the start of the branch. (And the commit message can have the answer to my question about why it's needed.)